cmake/modules/SetUpMacOS.cmake

```diff
@@ -85,6 +85,17 @@ if (CMAKE_SYSTEM_NAME MATCHES Darwin)
 
      set(CMAKE_C_LINK_FLAGS "${CMAKE_C_LINK_FLAGS} -bind_at_load -m64")
      set(CMAKE_CXX_LINK_FLAGS "${CMAKE_CXX_LINK_FLAGS} -bind_at_load -m64")
+     
+     if(asan)
+       # See also core/sanitizer/Readme.md for what's happening.
```

Comment:
Usually README name of file is capitalized, can you please rename it everywhere (name of file and in the comments)? Thanks!